packages/react-core/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -188,6 +188,8 @@ export const ToolbarItem: React.FunctionComponent<ToolbarItemProps> = ({
   ...props
 }: ToolbarItemProps) => {
   if (variant === ToolbarItemVariant.separator) {
+    // TODO: consider removing spread props here so we can update Divider role prop to union of
```

Comment:
Do we have an issue for this? Otherwise should we remove this comment?

Reply:
I updated some logic to have the union type on the new prop so removed this comment